packages/cozy-client/types/withMutation.d.ts

```
@@ -1,8 +1,5 @@
 export default withMutation;
-export type Wrapper = React.Component<any, any, any>;
-/**
- * @typedef {Component} Wrapper
- * @returns {Function}
- */
-declare function withMutation(mutation: any, options?: {}): Function;
-import React from "react";
+declare function withMutation(mutation: any, options?: {}): (WrappedComponent: any) => {
```

ESLint

[error] Replace `mutation:·any,·options?:·{}):·(WrappedComponent:·any` with `⏎··mutation:·any,⏎··options?:·{}⏎):·(⏎··WrappedComponent:·any⏎`

(prettier/prettier)